### ISSUE TITLE
With two files and three lines, the second file is shown in a single line

### ISSUE THREAD

**Issue by martinwguy** [martinwguy/xvi#148]
Repeat by opening two files and shrinking the window to three lines, which gives:
```
/* Copyright (c) 1990,1991,1992,1993 Chris and John Downey */
"src/alloc.c" 403/8430                                                         
/* Copyright (c) 1990,1991,1992,1993 Chris and John Downey */
```
or, if you shrink the window to two lines then enlarge it to three:
```
/* Copyright (c) 1990,1991,1992,1993 Chris and John Downey */
"src/alloc.c" 403/8430                                                         
"src/altstack.c" 99/2202  
```
In both of these cases, there is not room to show the second file, and only the first should be displayed in all 3 lines.

**Comment by martinwguy** [comment]
Working on...

**Comment by martinwguy** [comment]
Fixed by c4a4a0c1
